transmart-copy/src/main/groovy/org/transmartproject/copy/Database.groovy

```diff
@@ -68,25 +65,19 @@ class Database {
     final Connection connection
     final JdbcTemplate jdbcTemplate
     final NamedParameterJdbcTemplate namedParameterJdbcTemplate
-    final CopyManager copyManager
 
-    Database(boolean connectAsAdmin) {
-        String host = getenv('PGHOST')
+    Database(Map<String, String> params) {
+        String host = params['PGHOST']
         if (!host || host == '/tmp') {
             host = 'localhost'
         }
-        String port = getenv('PGPORT') ?: '5432'
-        String database = getenv('PGDATABASE') ?: 'transmart'
+        String port = params['PGPORT'] ?: '5432'
+        String database = params['PGDATABASE'] ?: 'transmart'
         String url = "jdbc:postgresql://${host}:${port}/${database}"
         String username
         String password
-        if (connectAsAdmin) {
```

Comment:
For use with the `vars` file from `transmart-data` it is important to distiguish the pairs `$PGUSER` with accompanying `$PGPASSWORD` and the user `tm_cz` with accompanying password `$TM_CZ_PWD` (defaults to `tm_cz`).

Reply:
I see. It looks like program required one execution with `--admin` flag and other without anyway. Now the same could be achieved by running the command with changed env. variables. `PGPASSWORD=tm_cz PGPASSWORD =TM_CZ_PWD <comand>`